modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -0,0 +1,139 @@
+    matchLabels:
+      cnf-worker-tuning: enabled
+  kubeletConfig:
+    cpuManagerPolicy: "static"
+    cpuManagerReconcilePeriod: "5s"
+    reservedSystemCPUs: "0,1"
+    memoryManagerPolicy: "Static"
+    evictionHard:
+      memory.available: "100Mi"
+    kubeReserved:
+      memory: "512Mi"
+    reservedMemory:
+      - numaNode: 0
+  limits:
+    memory: "1124Mi"
+    systemReserved:
+      memory: "512Mi"
+    topologyManagerPolicy: "single-numa-node" <1>
+    topologyManagerScope: "pod" <2>
+----
+<1> `topologyManagerPolicy` must be set to `single-numa-node`.
+<2> `topologyManagerScope` must be set to `pod`.
+
+
+.. Create the `KubeletConfig` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-kubeletconfig.yaml
+----
+
+. Create the `MachineConfigPool` custom resource that enables custom kubelet configurations for worker nodes:
+
+.. Save the following YAML in the `nro-machineconfig.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  labels:
+    cnf-worker-tuning: enabled
+    machineconfiguration.openshift.io/mco-built-in: ""
+    pools.operator.machineconfiguration.openshift.io/worker: ""
+  name: worker
+spec:
+  machineConfigSelector:
+    matchLabels:
+      machineconfiguration.openshift.io/role: worker
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker: ""
+----
+
+.. Create the `MachineConfigPool` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-machineconfig.yaml
+----
+
+. Create the `NUMAResourcesScheduler` custom resource that deploys the NUMA-aware custom pod scheduler:
+
+.. Save the following YAML in the `nro-scheduler.yaml` file:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1alpha1
+kind: NUMAResourcesScheduler
+metadata:
+  name: numaresourcesscheduler
+spec:
+  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-container-rhel8:v4.10"
+----
+
+.. Create the `NUMAResourcesScheduler` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-scheduler.yaml
+----
+
+.Verification
+
+Verify that the required resources deployed successfully:
+
+[source,terminal]
+----
+$ oc get all -n openshift-numaresources
+----
+
+.Example output
+[source,terminal]
+----
+NAME                                                    READY   STATUS    RESTARTS   AGE
+pod/numaresources-controller-manager-7575848485-bns4s   1/1     Running   0          13m
+pod/numaresourcesoperator-worker-dvj4n                  2/2     Running   0          16m
+pod/numaresourcesoperator-worker-lcg4t                  2/2     Running   0          16m
+pod/secondary-scheduler-56994cf6cf-7qf4q                1/1     Running   0          16m
+NAME                                          DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                     AGE
+daemonset.apps/numaresourcesoperator-worker   2         2         2       2            2           node-role.kubernetes.io/worker=   16h
```

Comment:
nit: just to be synced with the operator pods, let's adjust the daemonset Age to 16m instead of 16h